### ISSUE TITLE
Bug in make-project

### ISSUE THREAD

**Issue by thesps** [ipbus/ipbb#61]
The `RuntimeError` printing didn't work properly when I encountered such an error:
```
$ ipbb vivado make-project
make-project |
make-project | ----------------------------------------
make-project |
make-project | ****** Vivado v2019.2 (64-bit)
make-project |   **** SW Build 2708876 on Wed Nov  6 21:39:14 MST 2019
make-project |   **** IP Build 2700528 on Thu Nov  7 00:09:20 MST 2019
make-project |     ** Copyright 1986-2019 Xilinx, Inc. All Rights Reserved.
make-project |
make-project | Vivado% quit
make-project | quit
make-project | INFO: [Common 17-206] Exiting Vivado at Mon Mar 16 15:37:52 2020...
make-project | ----------------------------------------
ERROR ('TypeError' exception caught): 'can only join an iterable'

File "/home/sioni/Work/p2fwk-work/ipbb/src/ipbb/cmds/vivado.py", line 116, in makeproject
   "Error caught while generating Vivado TCL commands:\n" + "\n".join(lExc),
```

Changing the line from
`"Error caught while generating Vivado TCL commands:\n" + "\n".join(lExc),`
to
`"Error caught while generating Vivado TCL commands:\n" + str(lExc),`
fixed the problem in my case, but I'm not sure if there are situations where a list of errors are returned.

**Comment by alessandrothea** [comment]
Thanks Sioni.
Your fix looks good. I'm not sure where the `.join` comes from.
A relic from the past most likely.

**Comment by alessandrothea** [comment]
Fixed by #62
